GNUmakefile

```diff
@@ -12,7 +12,7 @@ test: fmtcheck
 	go test -i $(TEST) || exit 1
 	cd vcd ; VCD_SHORT_TEST=1 go test -v . -timeout 3m
 
-testacc: fmtcheck
+testacc: fmtcheck download-ova
```

Comment:
Is it necessary to run this operation in the Makefile?
I think the right place would be in function `TestMain` (`config_test.go`).
The way it is configured now, the test will work with `make testacc` bit will fail when we run the test as `go test` inside the `vcd` directory.

Reply:
Well, that was Linas idea and discussed here https://github.com/terraform-providers/terraform-provider-vcd/issues/137. From your point of view maybe is better to have in code - yet mentioned issue is `go test` is solved now with 
`if vcdShortTest {
	t.Skip(acceptanceTestsSkipped)
	return
}`
@lvirbalas We need common agreement should I invest time and move code from make files to go `config_test.go`

Reply:
@dataclouder the download logic is in the script right now, is your suggestion to rewrite it in Golang?

Reply:
Yes. Or else we have a test that can only run through `make` and not using `go test`.

Reply:
My bad. I didn't realise we're making the tests only `make` compatible with this.

@vbauzysvmware Yes, please invest time to refactoring this. The middle way would be to call the download script from Go, but you decide which is more elegant.

Reply:
Changes are done.